.github/workflows/unittests.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Test and Lint
```

Comment:
we don't do linting, or do we?

Reply:
Good point, I'll rename. I tried running black and flake8 but passing either would require lots of code changes